unit_tests/src/test_atomic_apply_acl_operations.cpp

```diff
@@ -575,8 +575,11 @@ TEST_CASE("Atomic acls respect entity zone")
     REQUIRE(rc_atomic_apply_acl_operations(static_cast<rcComm_t*>(conn),
                                            failing_json_input.c_str(),
                                            &json_error_string) == SYS_INVALID_INPUT_PARAM);
+    json parsed_error_string = json::parse(json_error_string);
     err_strs.push_back(json_error_string);
```

Comment:
I *think* the `json_error_string` would leak here if `json::parse` fails (because it will `throw`: https://json.nlohmann.me/api/basic_json/parse/#exceptions). Maybe just swap the order of these two lines:
```suggestion
    err_strs.push_back(json_error_string);
    json parsed_error_string = json::parse(json_error_string);
```

I think this is safe because of the exception safety guarantee outlined in the `nlohmann::json::parse` documentation linked above:
> Strong guarantee: if an exception is thrown, there are no changes in the JSON value.